runtime/src/iree/hal/drivers/cuda/dynamic_symbols.c

```diff
@@ -47,13 +82,13 @@ iree_status_t iree_hal_cuda_dynamic_symbols_initialize(
   memset(out_syms, 0, sizeof(*out_syms));
   iree_status_t status = iree_dynamic_library_load_from_files(
       IREE_ARRAYSIZE(kCUDALoaderSearchNames), kCUDALoaderSearchNames,
-      IREE_DYNAMIC_LIBRARY_FLAG_NONE, host_allocator,
-      &out_syms->loader_library);
+      IREE_DYNAMIC_LIBRARY_FLAG_NONE, host_allocator, &out_syms->cuda_library);
   if (iree_status_is_not_found(status)) {
     iree_status_ignore(status);
-    return iree_make_status(
-        IREE_STATUS_UNAVAILABLE,
-        "CUDA runtime library not available; ensure installed and on path");
+    IREE_RETURN_AND_END_ZONE_IF_ERROR(
+        z0, iree_make_status(IREE_STATUS_UNAVAILABLE,
+                             "CUDA runtime library not available; ensure "
+                             "installed and on path"));
```

Comment:
Readability wise it's preferable to make this a return so that users scanning the code can easily identify return points - the macro is conditional and requires the user to read back to see that it is in fact always a return.
```suggestion
    IREE_TRACE_ZONE_END(z0);
    return iree_make_status(IREE_STATUS_UNAVAILABLE, "CUDA runtime library not available; ensure installed and on path");
```